src/components/Article/index.tsx

```diff
@@ -11,8 +11,13 @@ export const Article = ({ article, onCommentsClick }: ArticleProps) => {
   return (
     <div className="h-dyn-screen snap-center flex flex-col justify-end items-center relative max-w-[500px] mx-auto">
       <Image article={article} />
-      <div className="from-black/50 to-transparent bg-gradient-to-t p-3 pb-8 max-w-screen">
-        <div className="w-[calc(100%-0.75rem)] max-w-screen grid grid-cols-[90%_10%] gap-3 box-">
+      <div className={`
```

Comment:
Please keep it a single line to maintain the code standard, I prefer making it that way and if the developer wants, they can activate wrap in their editor